theano/gpuarray/opt.py

```diff
@@ -340,29 +330,28 @@ def apply(self, fgraph):
 
             new_ops = None
             outputs = []
-            ex_opt_time = None
             # Apply the lifter
             for lopt in (self.local_optimizers_all +
                          self.local_optimizers_map.get(type(node.op), []) +
                          self.local_optimizers_map.get(node.op, [])):
```

Comment:
small optimization, I would change the order to be from the more specific to the more general (time both version just to be sure it don't slow things down. But this is my guess.

self.local_optimizers_map.get(node.op, []) + self.local_optimizers_map.get(type(node.op), []) + self.local_optimizers_all